app/controllers/flows_controller.rb

```diff
@@ -626,9 +627,24 @@ def self.ny_attributes(first_name: 'Testuser', last_name: 'Testuser')
         df_data_imported_at: 2.hours.ago,
       )
     end
+    def self.get_az_xml_sample(filing_status)
+      xml_service = StateFile::XmlReturnSampleService.new
+      case filing_status
+      when :single
+        xml_service.read("az_tycho_loanded")
+      when :married_filing_jointly
+        xml_service.read("az_martha_v2")
+      when :qualifying_widow
+        xml_service.read("az_leslie_qss_v2")
+      when :married_filing_separately
+        xml_service.read("az_sherlock_mfs")
+      when :head_of_household
+        xml_service.read("az_alexis_hoh_w2_and_1099")
+      end
+    end
```

Comment:
Because these are hardcoded, I could see an argument for moving this logic into `XmlReturnSampleService`. Something like `StateFile::XmlReturnSampleService.new.sample_for(state_code, filing_status)`. If we're moving to a world where there's a similar case statement for each active state and a case for each supported filing status, I'd say move that logic into `XmlReturnSampleService` now. If not now, then the very next time we add a state to the flow explorer.